designer/server/src/main/scala/pl/touk/nussknacker/ui/process/periodic/SchedulingDependencies.scala

```diff
@@ -0,0 +1,52 @@
+package pl.touk.nussknacker.ui.process.periodic
+
+import pl.touk.nussknacker.engine.api.component.{ComponentAdditionalConfig, DesignerWideComponentId}
+import pl.touk.nussknacker.engine.api.deployment.ProcessActionId
+import pl.touk.nussknacker.engine.api.process.ProcessingType
+import pl.touk.nussknacker.ui.db.DbRef
+import pl.touk.nussknacker.ui.process.deployment.ActionService
+import pl.touk.nussknacker.ui.process.repository.FetchingProcessRepository
+
+import java.util.function.Supplier
+import scala.concurrent.{ExecutionContext, Future}
+
+final class SchedulingDependencies(
+    val dbRef: DbRef,
+    val actionService: ProcessingTypeActionService,
+    val fetchingProcessRepository: FetchingProcessRepository[Future],
+    val configsFromProvider: Map[DesignerWideComponentId, ComponentAdditionalConfig]
+)
+
+trait ProcessingTypeActionService {
```

Comment:
It is moved from deployment-manager-api here, because it is related with scheduling mechanism only